README.md

```diff
@@ -0,0 +1,219 @@
+
+#### Don't withhold feedback for me
+
+Feedback is equally important for me to receive. Positive feedback helps me to validate my methods and, because I'm a human, it also feels good. Constructive feedback gives me insight into how I can improve. Regardless of the type of feedback you may have for me, know that I view it as a gift and am eager to have it. I typically try to solicit your feedback directly on a wide range of topics in our 1:1s but I may not always hit upon an area where you have feedback for me. Because of this, it's important that you take the initiative as necessary. Getting your feedback in a timely manner is also quite important. If something isn't going well, I prefer to make a change quickly rather than to have the problem fester.
+
+### Balancing life and work
+
+I believe that it's important to find a healthy balance between work and the rest of life. I want this balance for both of us.
+
+There's so much more to living a fulfilling life than success at work. Let's work hard while we're here and focus on life when we're not. To make you more comfortable about pursuing this balance, I'll endeavor to always model appropriate behavior. That means that you should not encounter me doing the following things*:
+
+- Staying late in the office; I usually head home by 6pm
+- Working on weekends
+- Using email or Slack off-hours
+- Scheduling meetings very early or very late in the day
+
+Given that I try to not do these myself, it would be unfair of me to expect you to. If you find me not leading by example --by exhibiting the above (or other) behaviors or encouraging behavior from you that leads to poor balance of work with life-- please voice your objection. The most likely explanation is that I did not consciously realize I was doing this and speaking up will help me to correct course.
+
+I prefer people to work smarter and not harder. If you find that you're needing to work well beyond 40 hours to be successful in your role, we should have a conversation about what adjustments can be made to make this not the case.
+
+_*The obvious exception here is emergency situations._
+
+## Communicating With Me
+
+My primary guideline for communication is to consider what channel is most appropriate for our interaction. If you're looking for more than five minutes of back-and-forth, let's talk in person. If you have a quick question, Slack is great. Just trying to disseminate some non-urgent information to a broad group? Email seems like a solid choice. Here's a few other details about how I operate with each channel:
```

Comment:
>Here _are_ a few other details

typo

Reply:
Great catch! The Authoritative Grammarian salutes you!